xbmc/interfaces/legacy/Player.cpp

```diff
@@ -426,26 +426,26 @@ namespace XBMCAddon
       }
     }
 
-    std::vector<String>* Player::getAvailableSubtitleStreams()
+    std::vector<String> Player::getAvailableSubtitleStreams()
```

Comment:
@jimfcarroll What do you mean? They were freed were they not? Otherwise we have lots of leaks..